circuits/cpp/src/aztec3/circuits/abis/public_circuit_public_inputs.hpp

```diff
@@ -104,9 +104,7 @@ template <typename NCT> struct PublicCircuitPublicInputs {
 
         std::vector<fr> inputs;
 
-        // NOTE: we omit the call_context from this hash function, and instead hash it within CallStackItem, for
-        // efficiency, so that fewer hashes are needed to 'unwrap' the call_context in the kernel circuit.
-        // inputs.push_back(call_context.hash());
```

Comment:
@iAmMichaelConnor I checked and the call context was **not** being included in the call stack item. I opted to include it here to avoid forgetting it again, but lmk if it's not the right approach.

Reply:
Thanks! This is likely an out of date comment from a while ago. I'm happy to reinstate it and optimise later :)